xml/geo_manage_i.xml

```diff
@@ -27,131 +27,164 @@
   command line client or with &hawk2;.
  </para>
  <sect1 xml:id="sec.ha.geo.manage.cli">
-  <title>Managing Tickets from Command Line</title>
-
-  <para>
-   Use the <command>booth&nbsp;client</command> command line tool to grant,
-   list, or revoke tickets as described in
-   <xref linkend="vl.ha.booth.client.cmds"/>. The
-   <command>booth&nbsp;client</command> commands can be run on any machine in
-   the cluster, not only the ones having the &boothd; running. The
-   <command>booth&nbsp;client</command> commands try to find the
-   <quote>local</quote> cluster by looking at the booth configuration file and
-   the locally defined IP addresses. If you do not specify a site which the
-   booth client should connect to (using the <option>-s</option> option), it
-   will always connect to the local site.
+  <title>Managing Tickets From Command Line</title>
+  <para> Use the <command>booth</command> command line tool to grant, list, or
+   revoke tickets as described in <xref linkend="sec.ha.geo.manage.cli.booth"/>.
   </para>
+  <warning>
+   <title><command>crm_ticket</command> and
+    <command>crm&nbsp;site&nbsp;ticket</command></title>
+   <para> If the booth service is not running for any reasons, you can also
+    manage tickets manually with <command>crm_ticket</command> or
```

Comment:
How about "purely manually" or "fully manually", to differentiate from the management of manual tickets?